app/controllers/miq_ae_class_controller.rb

```diff
@@ -468,6 +472,24 @@ def edit_class
     replace_right_cell
   end
 
+  def edit_class_record
+    assert_privileges("miq_ae_class_edit")
+    unless params[:id]
+      obj = find_checked_items
+      @_params[:id] = obj[0]
+    end
+    @hide_bottom_bar = true
+
+    class_rec = MiqAeClass.find(params[:id])
+    
+    render :json => {
+      :fqname        => class_rec.fqname,
+      :name        => class_rec.name,
+      :display_name => class_rec.display_name,
+      :description => class_rec.description
```

Comment:
Minor, but align the hash rockets:

```suggestion
      :fqname       => class_rec.fqname,
      :name         => class_rec.name,
      :display_name => class_rec.display_name,
      :description  => class_rec.description
```

Reply:
Done